pkg/router/controller/unique_host.go

```diff
@@ -84,6 +84,11 @@ func (p *UniqueHost) HandleEndpoints(eventType watch.EventType, endpoints *kapi.
 	return p.plugin.HandleEndpoints(eventType, endpoints)
 }
 
+// HandleNode processes watch events on the Node resource and calls the router
+func (p *UniqueHost) HandleNode(eventType watch.EventType, node *kapi.Node) error {
+	return p.plugin.HandleNode(eventType, node)
+}
+
```

Comment:
FYI, depending on which PR merges first - https://github.com/openshift/origin/pull/11201 - will need to add HandleNode to the `host_admitter` plugin.